web-app/src/components/CreateUser/container.js

```diff
@@ -17,31 +25,140 @@ export default Wrapped =>
 
     initialFormState() {
       return {
-        forename: '',
-        surname: '',
-        email: '',
-        password: '',
-        country: 1,
-        status: employeeStatus.ACTIVE, //TODO Create enum file
-        employeeRole: 1,
-        startDate: moment(),
+        forename: {
```

Comment:
Is this used elsewhere also? If so, it would be nice to have this stored elsewhere and imported. A central file holding the form data possibly.